inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp

```diff
@@ -534,6 +534,23 @@ void MKLDNNQuantizeNode::execute(mkldnn::stream strm) {
 void MKLDNNQuantizeNode::appendPostOps(mkldnn::post_ops& ops) {
     if (!isPostOpDataInitialized) {
         isPostOpDataInitialized = true;
+
+        // MKLDNN quantization_injectors assumes that quantization data memory is always aligned on 16 (by vector length)
+        // Otherwise it can lead to buffer over-read and performance penalties due to denormals.
+        size_t bufferAlignment = 16;
```

Comment:
```suggestion
        const size_t bufferAlignment = 16;
```

Reply:
Why we need this change?

Reply:
just minor comment
looks more correctly use const for unchanged variable